proto/rill/runtime/v1/expressions.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package rill.runtime.v1;
+
+import "google/protobuf/struct.proto";
+
+message Expression {
+  oneof expression {
+    google.protobuf.Value value = 1;
+    string column = 2;
+    ConditionExpression condition_expression = 3;
+  }
+}
+
+message ConditionExpression {
+  Operation operation = 1;
+  repeated Expression operands = 2;
+}
+
+enum Operation {
+  OPERATION_UNSPECIFIED = 0;
+  OPERATION_EQUALS = 1;
+  OPERATION_NOT_EQUALS = 2;
+  OPERATION_LESSER = 3;
+  OPERATION_LESSER_OR_EQUALS = 4;
+  OPERATION_GREATER = 5;
+  OPERATION_GREATER_OR_EQUALS = 6;
```

Comment:
Is it more common to use "equals" or "equal"? 

Also, should we consider the usual shorthands, like `EQ`, `NEQ`, `LT`, `LTE`, etc.?